kiota_abstractions/request_information.py

```diff
@@ -88,14 +88,29 @@ def url(self, url: Url) -> None:
 
     @property
     def request_headers(self) -> Optional[Dict]:
-        return self.headers
-
-    def add_request_headers(self, headers_to_add: Optional[Dict[str, str]]) -> None:
+        final = {}
+        for key, value in self.headers.items():
+            # Use set to remove duplicates
+            final[key] = ', '.join(list(set(value)))
```

Comment:
```suggestion
            final[key] = ', '.join(set(value))
```

This might change to 
```python
 final[key] = ', '.join(value)
```

if we choose to use a set for the map values.

Reply:
Updated in https://github.com/microsoft/kiota-abstractions-python/pull/39/commits/4d22efd4a9bdeae2be2067e9deb20e873d4631bb